packages/create-block/lib/utils.js

```diff
@@ -0,0 +1,12 @@
+const startCase = ( input ) => {
```

Comment:
Is it a goal to avoid a dependency on Lodash?

- https://lodash.com/docs/4.17.15#startCase
- https://lodash.com/docs/4.17.15#upperFirst

There's also per-method packages as an option, as a compromise option between bringing in the full dependency vs. maintaining these implementations ourselves.

https://lodash.com/per-method-packages

They're documented as discouraged. There's a bit more advice at that page on the broader problem we might be trying to address here.

Reply:
Yes, I figured out it isn't that much work to build those methods and avoid using `lodash`. In general, you want to install this package as fast as possible. However, we use a number of dependencies which probably bring `lodash` so we can use it if that is the case 🤣

Reply:
`lodash` is one of the dependencies of dependencies so let's use it instead :)